### PR TITLE
ios build and upload

### PR DESCRIPTION
**Pull request by floridemai**
Issue: https://github.com/nearform/polaris/issues/102

Created a new flow which builds and release both ios and android (Android was working fine, just ios was added)
Moved android upload to a new flow and kept the old e2e code (needs to be updated?... i think)

You can check the manually triggered flows that were successful here: https://github.com/nearform/polaris/actions/workflows/e2e.android.yml

Made the flow to be triggered when a new tag is created or manually.

### CODE REVIEW

File: package.json
Comment (simoneb) [inline package.json:15]: this looks good because you are providing these CLI args in https://github.com/nearform/polaris/pull/233/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R93. The only thing I don't like is it introduces an inconsistency between the android script, which provides them, and the ios one, which now doesn't. I would try to keep them conceptually similar, maybe removing the CLI args from the android script and providing them inside the workflow file instead, as you're doing with the ios one.

**Review by simoneb — APPROVED** [review]
LGTM

File: package.json
Comment (floridemai) [reply package.json:15]: 👍 Didn't touch the prev code, but makes sense to be consistent. I'll merge like this and will fix after this is merged to master. If in the future someone wants to build locally they'll just have to update this...

**Pull request merged**
